docs/user/advanced_features.rst

```diff
@@ -1243,6 +1244,50 @@ computation time and complex setup.
 	| Avg Mechanical Power (kW)  |      N/A     |    300      |    241     |
 	+----------------------------+--------------+-------------+------------+
 
+.. _control-reactive-with-PTO:
+
+Reactive Control with Direct Drive Power Take-Off
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The previous controllers only considered the mechanical power output. Although maximization
+of mechanical power allows for the maximum energy transfer from waves to body, it often does 
+not lead to maximum electrical power. The previous controller examples demonstrate the 
+controller types and energy transfer from waves to body, but the important consideration of 
+electrical power requires a PTO model. This example applies a reactive controller to the 
+sphere body with a simplified direct drive PTO model to maximize electrical power. Within 
+the Simulink subsystem for determining the PTO force, the controller prescribes the ideal or 
+desired force which is fed into the direct drive PTO. The current in the generator is then 
+used to control the applied force. 
+
+.. figure:: /_static/images/piPTOSimulink.png
+   :width: 500pt 
+   :align: center
+
+The PTO parameters used for this example are defined in the wecSimInputFile and correspond to 
+the Allied Motion Megaflux Frameless Brushless Torque Motors–MF0310 :cite:`allied`. The 
+results for the applied gains from Section :ref:`control-reactive` are shown in the figure 
+below. Mechanical power maximization requires significant net input electrical power 
+(signified by red bar - negative power is power harvested from the system). Thus, 
+instead of harvesting electrical power, power would need to be used to achieve mechanical 
+power maximization. 
+
+.. figure:: /_static/images/reactiveWithPTOCC.png
+   :width: 300pt 
+   :align: center
+
+On the other hand, by testing different controller gains, the gains which optimize for 
+maximum electrical power can be found as shown below. Increasing the proportional gain 
+and decreasing the integral gain magnitude leads to a maximum electrical power harvest 
```

Comment:
Minor comment: Agree with how the changing gains leads to better electrical power, but feel the statement would be more impactful if you can add a sentence or two on why those modifications led to the improved electrical power. For example was the electric generator able to run under more efficient conditions or more that the generally efficiency curve of the electric generator results in a greater proportion of energy input than harvested.